opentelemetry-otlp/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## vNext
 
+- `OtlpHttpClient.shutdown` `TonicLogsClient.shutdown`, and `TonicTracesClient.shutdown` now explicitly return a result. The
+    semantics of the method have not changed, but you will have a new lint encouraging you to consume these results.
```

Comment:
the 2nd sentence maybe omitted as these are anyway expected to be called from upstream sdk.